tests/Resources/config/packages/db_tools_deprecated_v2_conflict.yaml

```diff
@@ -0,0 +1,9 @@
+#
+# Deprecated but yet supported configuration options in ^2.0.
```

Comment:
Oups wrong comment, those are options that will raise exceptions (thay cannot be converted to legacy since the feature doesn't work the same way anymore, eg. binaries are not on per-vendor basis, but on connection-basis).